server/src/modules/slide-objects/ws.objects.service.ts

```diff
@@ -0,0 +1,89 @@
+import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model } from 'mongoose';
+import {
+  SlideObject,
+  SlideObjectDocument,
+} from 'src/schemas/slide-object.schema';
+import { UsersService } from '../users/users.service';
+import { SlidesService } from '../slides/slides.service';
+import { SlideObjectResponseObject } from '../../shared/interfaces/response-objects/SlideObjectResponseObject';
+import {
+  ObjectDataProps,
+  ObjectDataPropsWithId,
+} from '../../gateway/gateway.dto';
+import { WsException } from '@nestjs/websockets';
+
+@Injectable()
+export class WsObjectsService {
```

Comment:
`Object` is painfully generic, why not `SlideObject`?

Reply:
i mean its not using Object type or sth, the name is shorter and since there are boards, slides and objects it should be clear i guess..

Reply:
For me it's very unclear but whatever